pio/src/iSpindel.cpp

```diff
@@ -421,6 +421,7 @@ bool uploadData(uint8_t service)
     sender.add("temperature", Temperatur);
     sender.add("battery", Volt);
     sender.add("gravity", Gravity);
+    sender.add("interval", my_sleeptime);
```

Comment:
as @cwilling you mentioned, ubidots is ignorant to this challenge. I dont have a strong feeling here, but consider this additional data is billed.

Reply:
This comment about ubidots seems to be attached to code line 441 which is actually part of API_GENERIC. In my update to the PR, I removed extra interval field from API_UBIDOTS.

Reply:
correct, thanks